x-pack/plugins/security_solution/server/lib/timeline/routes/notes/get_notes.ts

```diff
@@ -48,7 +52,7 @@ export const getNotesRoute = (router: SecuritySolutionPluginRouter) => {
                 type: noteSavedObjectType,
                 search: docIdSearchString,
                 page: 1,
-                perPage: MAX_UNASSOCIATED_NOTES,
+                perPage: maxUnassociatedNotes,
```

Comment:
I know this was not the scope of this PR and should have been done when original code was added but i do not see any e2e or proper integration test for this API. I only see unit tests for this feature where almost everything seems to be mocked. Honestly, in my opinion that does not give too much confidence. Let me know what you think.

 If possible, consider adding api integration tests in `x-pack/test/security_solution_api_integration/`